### PR TITLE
update external tests to run with docker images provided by adoptium

### LINKED ISSUE #3116: update external test to run with docker images provided by adoptium
For external tests we support images defined by following variables. That is 10 * 2 * 8 * 2 * 2 * 20=12800. That is a huge number. As we moved to adoptium official adoptium images  of the Eclipse Temurin Distribution has been adjusted we will also adjust our supported images. 
```
# Supported versions
supported_versions="8 9 10 11 12 13 14 15 16 17"

# Supported JVMs
supported_jvms="hotspot openj9"

# Supported distros
supported_os="alpine debian debianslim ubi ubi-minimal centos clefos ubuntu"

# Supported packges
supported_packages="jdk jre"

# Supported builds
supported_builds="slim full"

# Supported tests
supported_tests="external_custom camel derby elasticsearch jacoco jenkins functional-test kafka lucene-solr openliberty-mp-tck payara-mp-tck quarkus quarkus_quickstarts scala system-test thorntail-mp-tck tomcat tomee wildfly wycheproof netty spring"
```

For now we will adjust to 

```
# Supported versions
supported_versions="8 11 17"

# Supported JVMs
supported_jvms="hotspot" (add openj9 later?)

# Supported distros
supported_os="ubuntu" (add alpine, windows, ubi later?)

# Supported packges
supported_packages="jdk jre"

# Supported builds
supported_builds="full"

# Supported tests
supported_tests="external_custom camel derby elasticsearch jacoco jenkins functional-test kafka lucene-solr openliberty-mp-tck payara-mp-tck quarkus quarkus_quickstarts scala system-test thorntail-mp-tck tomcat tomee wildfly wycheproof netty spring"
```

The definition is in https://github.com/adoptium/aqa-tests/blob/master/external/common_functions.sh.

Along with this test.properties under each third party application directory will need to update. For example test.properties in camel https://github.com/adoptium/aqa-tests/blob/master/external/camel/test.properties 

```
debian_packages="git maven"
 alpine_packages="bash git maven"
 centos_packages="git maven"
 ubi_packages="git maven"
```
will be replaced by

```
ubuntu_packages="git maven"
```

Function function set_test_info()  and set_external_custom_test_info() in https://github.com/adoptium/aqa-tests/blob/master/external/common_functions.sh will also need to update to make this work.

### PR DESCRIPTION
**Pull request by LizyBbethy**
Added cleanup and update of files to latest official adoptium images of the Eclipse Temurin Distribution

Fixes #3116
Signed-off by; LizyBbethy @LizyBbethy

### CODE REVIEW

**Comment by LizyBbethy** [comment]
hi @smlambert @sophia-guo  here is the new PR with changes and new grinder test jobs

jacoco Grinder at: https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/2556/
jacoco Grinder at ; https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/2557/   with jdk 11
quarkus_quickstarts Grinder at; https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/2559/
jenkins Grinder at; https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/2560/

**Review by smlambert — APPROVED** [review]
LGTM, thanks @LizyBbethy !

**Review by adamfarley — APPROVED** [review]
LGTM

**Review by Mesbah-Alam — APPROVED** [review]
LGTM!

File: external/wycheproof/test.properties [hunk 1]
```diff
@@ -4,7 +4,4 @@ home_path="\${WORKDIR}"
 test_results="testResults"
 tag_version="master"
 bazel_version="1.2.1"
-debian_packages="git wget unzip zip g++"
-alpine_packages="bash git wget unzip zip g++ bash"
-centos_packages="git wget unzip zip gcc-c++"
-ubi_packages="git wget unzip zip gcc-c++"
+ubuntu_packages="git wget unzip zip gcc-c++"
```
Comment (sophia-guo) [inline external/wycheproof/test.properties:7]: Did you run grinder changing ubuntu_packages from `git wget unzip zip g++` to  `git wget unzip zip gcc-c++`?

File: external/tomcat/test.properties [hunk 2]
```diff
@@ -4,7 +4,4 @@ home_path=""
 tag_version="10.0.7"
 ant_version="1.10.6"
 openssl_version="1.1.1d"
-debian_packages="git wget tar make gcc linux-libc-dev libc6-dev perl"
-alpine_packages="bash git wget tar make gcc libc-dev perl linux-headers"
-centos_packages="git wget tar make gcc glibc-devel perl"
-ubi_packages="git wget tar make gcc glibc-devel"
+ubuntu_packages="git wget tar make gcc glibc-devel"
```
Comment (sophia-guo) [inline external/tomcat/test.properties:7]: Same as following question any reason change the `git wget tar make gcc linux-libc-dev libc6-dev perl` to `git wget tar make gcc glibc-devel`?

File: external/scala/test.properties [hunk 3]
```diff
@@ -3,7 +3,4 @@ script="scala-test.sh"
 home_path=""
 tag_version="v2.13.2"
 sbt_version="1.2.8"
-debian_packages="git wget tar curl gpg gpg-agent"
-alpine_packages="bash git wget tar curl gnupg"
-centos_packages="git wget tar curl gpg"
-ubi_packages="git wget tar curl gpg"
+ubuntu_packages="git wget tar curl gpg"
```
Comment (sophia-guo) [inline external/scala/test.properties:6]: Same here, the reason change `git wget tar curl gpg gpg-agent` to `git wget tar curl gpg`?

File: external/netty/test.properties [hunk 4]
```diff
@@ -4,7 +4,4 @@ test_commands="mvn clean package"
 home_path="\${WORKDIR}"
 test_results="testResults"
 tag_version="4.1"
-debian_packages="git maven autoconf automake libtool make tar gcc-multilib libaio-dev"
-alpine_packages="bash git maven"
-centos_packages="git maven"
-ubi_packages="git maven"
+ubuntu_packages="git maven"
```
Comment (sophia-guo) [inline external/netty/test.properties:7]: same here, the reason change  `git maven autoconf automake libtool make tar gcc-multilib libaio-dev` to `git maven`?

File: external/wycheproof/test.properties (hunk 1 above)
Comment (LizyBbethy) [reply external/wycheproof/test.properties:7]: thanks for the suggestion , going to double check it too

Comment (LizyBbethy) [reply external/wycheproof/test.properties:7]: console
as per this grinder https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/2570/console great catch!
Clearing it up for all in a new branch and PR

**Pull request merged**